advisories/unreviewed/2023/11/GHSA-5wvp-7f3h-6wmm/GHSA-5wvp-7f3h-6wmm.json

```
@@ -1,23 +1,49 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-5wvp-7f3h-6wmm",
-  "modified": "2023-11-09T09:30:26Z",
+  "modified": "2023-11-09T09:30:33Z",
   "published": "2023-11-09T09:30:26Z",
   "aliases": [
     "CVE-2023-47248"
   ],
+  "summary": "PyArrow: Arbitrary code execution when loading a malicious data file",
   "details": "Deserialization of untrusted data in IPC and Parquet readers in PyArrow versions 0.14.0 to 14.0.0 allows arbitrary code execution. An application is vulnerable if it reads Arrow IPC, Feather or Parquet data from untrusted sources (for example user-supplied input files).\n\nThis vulnerability only affects PyArrow, not other Apache Arrow implementations or bindings.\n\nIt is recommended that users of PyArrow upgrade to 14.0.1. Similarly, it is recommended that downstream libraries upgrade their dependency requirements to PyArrow 14.0.1 or later. PyPI packages are already available, and we hope that conda-forge packages will be available soon.\n\nIf it is not possible to upgrade, we provide a separate package `pyarrow-hotfix` that disables the vulnerability on older PyArrow versions. See  https://pypi.org/project/pyarrow-hotfix/  for instructions.\n\n",
   "severity": [
 
   ],
   "affected": [
-
+    {
+      "package": {
+        "ecosystem": "PyPI",
+        "name": "pyarrow"
+      },
+      "ranges": [
+        {
+          "type": "ECOSYSTEM",
+          "events": [
+            {
+              "introduced": "0.14.0"
+            },
+            {
+              "fixed": "14.0.1"
+            }
+          ]
+        }
+      ],
+      "database_specific": {
+        "last_known_affected_version_range": "< 14.0.0"
```

Seems like I made a mistake here.
```suggestion
        "last_known_affected_version_range": "< 14.0.1"
```